ExomeGermlineSingleSample.inputs.json

```diff
@@ -0,0 +1,48 @@
+{
+  "ExomeGermlineSingleSample.sample_and_unmapped_bams": {
+    "final_gvcf_base_name": "HG00133.GRCh38DH.exome",
+    "unmapped_bam_suffix": "HG00133.GRCh38DH.exome",
+    "flowcell_unmapped_bams": ["gs://gatk-best-practices/exome-analysis-pipeline/SRR099969.unmapped.bam"],
+    "sample_name": "HG00133.alt_bwamem_GRCh38DH.20150826.GBR.exome",
+    "base_file_name": "HG00133.GRCh38DH.exome"
+  },
+
+  "ExomeGermlineSingleSample.references": {
+    "break_bands_at_multiples_of": 1000000,
+    "calling_interval_list": "gs://broad-references/hg38/v0/exome_calling_regions.v1.interval_list",
+    "dbsnp_vcf": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf",
+    "dbsnp_vcf_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf.idx",
+    "reference_fasta": {
+      "ref_fasta": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
+      "ref_fasta_index": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
+      "ref_dict": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+      "ref_alt": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.alt",
+      "ref_ann": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.ann",
+      "ref_sa": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.sa",
+      "ref_amb": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.amb",
+      "ref_bwt": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.bwt",
+      "ref_pac": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.pac"
+    },
+    "known_indels_sites_vcfs": [
+      "gs://broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz",
+      "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.known_indels.vcf.gz"
+    ],
+    "known_indels_sites_indices": [
+      "gs://broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi",
+      "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.known_indels.vcf.gz.tbi"
+    ],
+    "contamination_sites_ud": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.contam.UD",
+    "contamination_sites_bed": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.contam.bed",
+    "contamination_sites_mu": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.contam.mu",
+    "evaluation_interval_list": "gs://broad-references/hg38/v0/exome_evaluation_regions.v1.interval_list",
+    "haplotype_scatter_count": 50 
+  },
+
+  "ExomeGermlineSingleSample.target_interval_list": "gs://broad-references/hg38/v0/exome_calling_regions.v1.interval_list",
+  "ExomeGermlineSingleSample.bait_interval_list": "gs://broad-references/hg38/v0/exome_calling_regions.v1.interval_list",
```

Comment:
The repo is for public use so they probably shouldn't be using our bait/interval lists. I'll add the following to the ReadMe as an input requirement because commenting in the json has become difficult.
"Unique exome calling, target, and bait .interval_list obtained from sequencing provider. Generally the calling, target, and bait files will not be the same."
FYI , I'm using the same file for all of these.